examples/ActivityLogger/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using SoCreate.Extensions.Logging;
+using SoCreate.Extensions.Logging.ActivityLogger;
+
+namespace ActivityLogger
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            LoggerBootstrapper.InitializeServiceFabricRegistration(
+                (serviceName, configuration, addloggingToServiceContext) =>
```

Comment:
Rename parameter from "addloggingToServiceContext" to "addServiceContextToLogging".